integration-tests/flyway/src/main/java/io/quarkus/it/flyway/AppEntity.java

```diff
@@ -0,0 +1,51 @@
+package io.quarkus.it.flyway;
+
+import java.util.Objects;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+/**
```

Comment:
Missing some javadoc comment ?

Reply:
Hello @machi1990, that's just an entity used in test case so I hadn't bothered to add any javadoc. However, since I was anyway updating this PR, I decided to add a simple javadoc comment to this one. The PR has now been updated.

Reply:
Thanks @jaikiran, great work.